CONTRIBUTING.md

```diff
@@ -0,0 +1,12 @@
+## Making changes to CitGM
+#### There are a few basic requirements for making changes to CitGM
+
+* Consider creating an issue prior to submitting a PR as this can help speed up the process
+* Include tests for your code wherever possible
+* Include any documentation or man page changes where necessary
+* Ensure that `npm test` passes before submitting the PR
+* Please squash each logical change into a single commit
+* Follow the nodejs commit guidelines shown [here](https://github.com/nodejs/node/blob/master/CONTRIBUTING.md#step-3-commit)
```

Comment:
https://github.com/nodejs/citgm/pull/314#discussion_r98095427 is still outstanding

>Maybe worth just including the basic rules in a few bullet points